test/.eslintrc

```diff
@@ -2,3 +2,6 @@
   extends: ../.eslintrc
   env:
     mocha: true
+
+  rules:
+  	no-unused-vars: [2, {"var":"local", "args":"none"}]
```

Comment:
This is extra. See https://github.com/apigee-127/swagger-test-templates/pull/3/files#r33403397